### PR TITLE
New package: lumina-pdf-1.6.0

### PR DESCRIPTION
**Pull request by beanpole135**
This is the PDF viewer and presentation utility from the Lumina desktop.
It is a stand-alone tool written in Qt5. No dependency on the Lumina desktop itself.

Sponsored by: Project Trident

### CODE REVIEW

**Comment by abenson** [comment]
Please don't open/close PRs, it's mark it harder to track reviews. If you need to update the commit, use `git commit --amend` and `git push -f`

**Comment by beanpole135** [comment]
GitHub messed something up when I opened the first PR and pulled in the previous lumina-calculator commit/branch, with the lumina-pdf subject. That is why I closed it and re-opened to ensure it used the proper branch for the PR.

**Comment by abenson** [comment]
You can also use `git rebase -i HEAD~5` to interactively rebase and drop specific commits if they don't belong.

File: srcpkgs/lumina-pdf/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+makedepends="qt5-devel poppler-qt5-devel"
+short_desc="PDF reader and presentation utility from the Lumina Desktop"
+maintainer="Ken Moore <ken@project-trident.org>"
+license="BSD-3-Clause"
+homepage="https://github.com/lumina-desktop/lumina-pdf"
+distfiles="https://github.com/lumina-desktop/lumina-pdf/archive/v${version}.tar.gz"
+checksum=376f8a48b0c33bcf1603228748dd0d531eb3d59cda703a3927d950459674144b
+
+post_install() {
+	vlicense ${wrksrc}/LICENSE
```
Comment (Hoshpak) [inline srcpkgs/lumina-pdf/template:18]: Since the stage is executed in wrksrc, there should be no need to define it as part of the path here.

Comment (beanpole135) [reply srcpkgs/lumina-pdf/template:18]: Unfortunately, removing the "${wrksrc}" from post_install breaks the build. I think this is due to the custom "build_wrksrc" directory getting used for all [pre/post]_install() routines as well.

Comment (Hoshpak) [reply srcpkgs/lumina-pdf/template:18]: Alright then, I didn't realize that. I think we can leave it like that.

**Pull request merged**